ui/app/public/index.html

```diff
@@ -30,8 +30,16 @@
     href="/studio/static-assets/themes/cstudioTheme/css/font-awesome.min.css"
     rel="stylesheet"
   >
+  <link
+    href="/js/monaco/vs/editor/editor.main.css"
+    rel="stylesheet"
+  >
 </head>
 <body>
+<script>var require = { paths: { 'vs': '/js/monaco/vs' } };</script>
+<script src="/js/monaco/vs/loader.js"></script>
+<script src="/js/monaco/vs/editor/editor.main.nls.js"></script>
```

Comment:
What's this one for?

Reply:
to load monaco editor, it needs that files

Reply:
yes, but what is this one specifically for? (main.nsl.js)

Reply:
I think it contains the languages